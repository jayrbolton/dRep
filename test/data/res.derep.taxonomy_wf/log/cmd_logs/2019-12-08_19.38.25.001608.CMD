nucmer --mum -p /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/ANImf_files/Bin.034.fasta/Bin.034.fasta_vs_Bin.034.fasta -c 65 -g 90 /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.034.fasta /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.034.fasta; delta-filter -r -q /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/ANImf_files/Bin.034.fasta/Bin.034.fasta_vs_Bin.034.fasta.delta > /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/ANImf_files/Bin.034.fasta/Bin.034.fasta_vs_Bin.034.fasta.delta.filtered
