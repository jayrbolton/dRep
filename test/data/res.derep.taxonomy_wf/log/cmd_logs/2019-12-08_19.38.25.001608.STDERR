/bin/sh: 1: nucmer: not found
/bin/sh: 1: delta-filter: not found
